Clinical-Genomics/scout
# Use a FlaskForm with a CSRF token when passing form data to chanio-report

This is an attempt at fixing #2589 and #2897

I'm not sure how to reproduce the situation of these bugs and what I'd do is just giving this alternative code a try. If we don't get the error from the bugs above any more then the bug is fixed! 😄 

**How to test**:
1. Install on Scout stage and try to download the coverage report for a case (the case should have coverage in chanjo db), for instance [this one](https://scout-stage.scilifelab.se/cust000/NZ-LCH-lucigentest) or [this one](https://scout-stage.scilifelab.se/cust000/15035-miptest)

**Expected outcome**:
The functionality should be working and should return the same coverage for these samples as the coverage report produced by "main" branch
Take a screenshot and attach or copy/paste the output.

**Review:**
